app/views/giraffe/contribute.scala.html

```diff
@@ -24,7 +24,11 @@
     @if(inLandingPageTest) {
         @contributionsAlt(pageInfo, countryGroup, Some(maxAmount), Some("contributePage.js"), errorMessage)(block)
     } else {
-        @contributionsMain(pageInfo, countryGroup, Some(maxAmount), Some("contributePage.js"), errorMessage)(block)
+        @if(intCmpCode == Some("gdnwb_copts_memco_this_land_epic_bottom") || intCmpCode == Some("gdnwb_copts_memco_this_land_epic_top") || intCmpCode == Some("gdnwb_copts_memco_this_land_thrasher")) {
```

Comment:
just to clarify this I'd change the check to `intCmpCode.exists(_.contains("_this_land_"))`